src/pages/Stores/components/S3Browser/S3Browser.tsx

```diff
@@ -5,23 +5,30 @@ import './S3Browser.scss';
 import { Workspace } from '@/context/WorkspaceContext/types';
 import { useWorkspace } from '@/hooks/useWorkspace';
 
-const createStorageBrowserWithWorkspace = (activeWorkspace: Workspace) =>
-  createStorageBrowser({
+const createStorageBrowserWithWorkspace = (activeWorkspace: Workspace) => {
+  const locations = [];
+  activeWorkspace.stores.forEach((store) => {
```

Comment:
Probably worth making this `activeWorkspace?.stores`, similar to above, if its not set it'll crash the site